Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/ImageClassifier/ImageClassifier.xaml.cs

```diff
@@ -0,0 +1,448 @@
+                    { Classifier.ResNet,            "ms-appx:///LargeModels/resnet50-caffe2-v1-9.onnx"},
+                    { Classifier.VGG,               "ms-appx:///LargeModels/vgg19-7.onnx"},
+                    { Classifier.ZFNet512,          "ms-appx:///LargeModels/zfnet512-9.onnx"},
+                };
+            }
+
+            if (_postProcessorDictionary == null)
+            {
+                _postProcessorDictionary = new Dictionary<Classifier, Func<LearningModel>>{
+                    { Classifier.SqueezeNet,        () => TensorizationModels.SoftMaxThenTopK(TopK) },
+                    { Classifier.MobileNet,         () => TensorizationModels.SoftMaxThenTopK(TopK) },
+                    { Classifier.GoogleNet,         () => TensorizationModels.TopK(TopK) },
+                    { Classifier.DenseNet121,       () => TensorizationModels.ReshapeThenSoftmaxThenTopK(new long[] { BatchSize, _imagenetLabels.Count, 1, 1 },
+                                                                                                    TopK,
+                                                                                                    BatchSize,
+                                                                                                    _imagenetLabels.Count) },
+                    { Classifier.Inception_V1,      () => TensorizationModels.TopK(TopK) },
+                    { Classifier.Inception_V2,      () => TensorizationModels.TopK(TopK) },
+                    { Classifier.ShuffleNet_V1,     () => TensorizationModels.TopK(TopK) },
+                    { Classifier.ShuffleNet_V2,     () => TensorizationModels.SoftMaxThenTopK(TopK) },
+                    { Classifier.EfficientNetLite4, () => TensorizationModels.SoftMaxThenTopK(TopK) },
+                    // Large Models
+                    { Classifier.AlexNet,           () => TensorizationModels.TopK(TopK) },
+                    { Classifier.CaffeNet,          () => TensorizationModels.TopK(TopK) },
+                    { Classifier.RCNN_ILSVRC13,     () => TensorizationModels.TopK(TopK) },
+                    { Classifier.ResNet,            () => TensorizationModels.SoftMaxThenTopK(TopK) },
+                    { Classifier.VGG,               () => TensorizationModels.SoftMaxThenTopK(TopK) },
+                    { Classifier.ZFNet512,          () => TensorizationModels.TopK(TopK) },
+                };
+            }
+
+            if (_preProcessorDictionary == null)
+            {
+                _preProcessorDictionary = new Dictionary<Classifier, Func<LearningModel>>{
+                    { Classifier.SqueezeNet,        null }, // No preprocessing required
+                    { Classifier.MobileNet,         () => TensorizationModels.Normalize0_1ThenZScore(224, 224, 4,
+                                                                                                new float[] { 0.485f, 0.456f, 0.406f },

```

Comment:
we should have comments explaining where these numbers come from. Also, maybe have links to the research papers where the values come from?

Reply:
@smk2007 may be able to answer this

Reply:
They come from the preprocessing steps described in the model zoo:
https://github.com/onnx/models/tree/master/vision/classification/mobilenet

Each model has its own preprocessing steps and values based on the model's definition.